packages/app-store/stripepayment/lib/client/createPaymentLink.ts

```diff
@@ -9,11 +9,12 @@ export function createPaymentLink(opts: {
   name?: Maybe<string>;
   date?: Maybe<string>;
   email?: Maybe<string>;
+  tz?: Maybe<string>;
   absolute?: boolean;
 }): string {
-  const { paymentUid, name, email, date, absolute = true } = opts;
+  const { paymentUid, name, email, date, tz, absolute = true } = opts;
   let link = "";
   if (absolute) link = WEBSITE_URL;
-  const query = stringify({ date, name, email });
+  const query = stringify({ date, name, email, tz });
```

Comment:
why do we need tz here?

Reply:
false assumption here. pushed the change removing this